irohad/pending_txs_storage/pending_txs_storage.cpp

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "pending_txs_storage/pending_txs_storage.hpp"
+
+#include "multi_sig_transactions/state/mst_state.hpp"
+#include "pending_txs_storage.hpp"
+#include "pending_txs_storage/pending_txs_storage.hpp"
+
+namespace iroha {
+
+  PendingTransactionStorage::PendingTransactionStorage(
+      StateObservable updatedBatches,
+      BatchObservable preparedBatch,
+      BatchObservable expiredBatch) {
+    updated_batches_subscription_ =
+        updatedBatches.subscribe([this](const SharedState &updatedBatches) {
+          this->updatedBatchesHandler(updatedBatches);
+        });
+    prepared_batch_subscription_ =
+        preparedBatch.subscribe([this](const SharedBatch &preparedBatch) {
+          this->preparedBatchHandler(preparedBatch);
+        });
+    expired_batch_subscription_ =
+        expiredBatch.subscribe([this](const SharedBatch &expiredBatch) {
+          this->expiredBatchHandler(expiredBatch);
+        });
+  }
+
+  PendingTransactionStorage::~PendingTransactionStorage() {
+    updated_batches_subscription_.unsubscribe();
+    prepared_batch_subscription_.unsubscribe();
+    expired_batch_subscription_.unsubscribe();
+  }
+
+  PendingTransactionStorage::SharedTxsCollectionType
+  PendingTransactionStorage::getPendingTransactions(
+      const AccountIdType &accountId) const {
+    std::shared_lock<std::shared_timed_mutex> lock(mutex_);
+    auto creator_it = storage_.index.find(accountId);
+    if (storage_.index.end() != creator_it) {
```

Comment:
could you revert the condition, so the code is more linirized, e.g:
```
if (storage_.index.end() == creator_it) {
    return {};
}

...
```

Reply:
I prefer leaving it as is because of two reasons:
1. Such change will introduce style inconsistency with the following code (line 49)
2. For me, it is a matter of taste :)

I can apply the change if you insist on it.